website/content/docs/security/acl/acl-system.mdx

```diff
@@ -202,16 +202,16 @@ the token itself.
 
 #### Builtin Tokens
 
-During cluster bootstrapping when ACLs are enabled both the special `anonymous` and the `master` token will be
+During cluster bootstrapping when ACLs are enabled both the special `anonymous` and the `initial_management` token will be
 injected.
 
 - **Anonymous Token** - The anonymous token is used when a request is made to Consul without specifying a bearer token.
   The anonymous token's description and policies may be updated but Consul will prevent this token's deletion. When created,
   it will be assigned `00000000-0000-0000-0000-000000000002` for its Accessor ID and `anonymous` for its Secret ID.
 
-- **Master Token** - When a master token is present within the Consul configuration, it is created and will be linked
-  With the builtin Global Management policy giving it unrestricted privileges. The master token is created with the Secret ID
-  set to the value of the configuration entry.
+- **Initial Management Token** - When an initial management token is present within the Consul configuration, it is created
+  and will be linked with the builtin Global Management policy giving it unrestricted privileges. The initial management
+  token is created with the Secret ID set to the value of the configuration entry.
```

Comment:
I think we should mention that this was previously called the "Master" token (in case someone is confused why the "master" token doesn't appear in the docs because they are on Consul 1.4-1.10).

```suggestion
- **Initial Management Token** - When an initial management token is present within the Consul configuration, it is created
  and will be linked with the builtin Global Management policy giving it unrestricted privileges. The initial management
  token is created with the Secret ID set to the value of the configuration entry.
  In Consul 1.4 - 1.10, this was called the `master` token. It was renamed to `initial_management` token in Consul 1.11.
```